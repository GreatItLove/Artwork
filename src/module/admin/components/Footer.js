import React from 'react';
import FooterNavgition from '../../menu/footer/components/FooterNavgition';

const Footer = () => (
  <footer className="readmin-footer">
    <FooterNavgition />
    <p>Copyright Â© ArtWork 2017</p>
  </footer>
);

export default Footer;
