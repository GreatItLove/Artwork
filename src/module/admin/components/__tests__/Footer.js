/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toSON from 'enzyme-to-json';

import Footer from '../Footer';

describe('Footer component', () => {
  it('Shallow snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(toSON(wrapper)).toMatchSnapshot();
  });
});
