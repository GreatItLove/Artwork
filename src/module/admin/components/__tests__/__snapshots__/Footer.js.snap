// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Shallow snapshot 1`] = `
<footer
  className="readmin-footer"
>
  <FooterNavgition />
  <p>
    Copyright Â© ArtWork 2017
  </p>
</footer>
`;
